### PR TITLE
MRP question on loss of nature (41.1)

### LINKED ISSUE #201: Aid Alliance MRP data (from HOPE not hate survey)
We have data about _one_ of the Aid Alliance questions from the recent HOPE not hate MRP poll (see shared Drive) – they’re also working on getting us data from their other questions.

### PR DESCRIPTION
**Pull request by alexander-griffen**
Fixes #201 
Adds the new MRP question to Zarino's existing import script
image

### CODE REVIEW

**Comment by zarino** [comment]
> I'm quite confused as the answers here don't seem to match up with the question. Am I missing something?

Oh, damn, yes, the answers should be "agree", "disagree", etc, as that’s what’s in the source spreadsheet’s column headers.

> Oh, also the filename is Aid Alliance but the credits are Hope Not Hate so I assume one of those details is wrong?

Yes and no. The question was _written_ by Aid Alliance, for Hope Not Hate to run in their MRP survey, which was conducted by Focaldata. Same as the other three MRP questions, which were _written_ by TCC, but run in Hope Not Hate’s survey. Not sure on the wording, but TCC were happy with how we described their existing three questions, so I assume they’ll be happy with this Aid Alliance one. Can always check and amend later.

File: hub/management/commands/import_hnh_polling_data.py [hunk 1]
```diff
@@ -44,6 +44,70 @@ class Command(BaseImportFromDataFrameCommand):
...
+                    "slug": "net_likely",
+                    "label": "Net likely",
+                },
+                {
+                    "source_string": "NET_unlikely",
+                    "slug": "net_unlikely",
+                    "label": "Net unlikely",
+                },
+            ],
+        },
```
[the remark this answers is not in the document]
Comment (alexander-griffen) [reply hub/management/commands/import_hnh_polling_data.py:111]: Oops! I got a little too copy and paste happy, I think

[the remark this answers is not in the document]
Comment (alexander-griffen) [reply hub/management/commands/import_hnh_polling_data.py:111]: This has been added in a fixup

**Comment by alexander-griffen** [comment]
@zarino Could you check the layout now there's a new dataset, please? :)

**Review by zarino — COMMENTED** [review]
Looks good to me :+1:

File: hub/management/commands/import_hnh_polling_data.py [hunk 2]
```diff
@@ -264,7 +372,10 @@ def extract_and_save_data(self):
                     data_types[col["slug"]] = data_type
 
             df = pd.read_csv(settings.BASE_DIR / "data" / file["source_filename"])
-
+            if file["data_set_name"] == "hnh_mrp_41-1":
```
Comment (zarino) [inline hub/management/commands/import_hnh_polling_data.py:375]: @alexander-griffen

1. If we’re skipping the first (empty) row for Question 41.1, shouldn’t we _also_ do it for Question 25 (which shares exactly the same formatting)?
2. The [pandas.read_csv() docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html) suggest`skip_blank_lines` defaults to `True`, so are you sure we need to skip the first (empty) row manually?

**Review by zarino — COMMENTED** [review]
FYI @alexander-griffen I’ve rebased your two commits, to move the "likely / not likely" fix from the second commit into the first, so that each commit is entirely about one question alone.

But while I was doing it, I spotted the `skiprows=1` condition on the first dataset, and how it wasn’t there for the second. Given your imports worked fine for both datasets, I have a feeling the default `skip_blank_lines` was being applied for Question 25’s CSV, which means we probably don’t need the condition at all for Question 41.1.

File: hub/management/commands/import_hnh_polling_data.py (hunk 2 above)
Comment (alexander-griffen) [reply hub/management/commands/import_hnh_polling_data.py:375]: I think you're right, this is unnecesary. I will remove it and rebase that in.

**Pull request merged**